### ISSUE TITLE
Add element center point helper

### ISSUE THREAD

**Issue by Fibrasek** [ruby2d/ruby2d#133]
First, awesome project!

Now to the issue, what about some nice helper to make it easier to get an element (`Text`, `Circle`...) center coordinates?

This seems to be more a `quality of life` helper when working with collision or simply when nesting elements, for example, some text inside a circle or a rectangle.

I can make this feature if you find it's worth.

**Comment by blacktm** [comment]
Thanks! That's a great idea. Every visible class in the window includes [`Renderable`](https://github.com/ruby2d/ruby2d/blob/v0.7.0/lib/ruby2d/renderable.rb), so maybe we add a `.center` method there to return the value.

**Comment by Fibrasek** [comment]
@blacktm awesome! I'll work on it then :)

**Comment by blacktm** [comment]
Sounds good. I was also thinking, maybe we should add a way to change where the x, y are actually positioned, like an offset or something. It’s not always useful for them to be the upper-left corner (I always have to transform them in my games).

**Comment by artob** [comment]
This would be very helpful, especially for centering text where variable-width fonts make it difficult to do by any other means.

**Comment by Fibrasek** [comment]
I've been very busy, could not write it yet, maybe in the next week I'll get back to it!

**Comment by Nakilon** [comment]
How do I align text?

**Comment by Nakilon** [comment]
Managed to mimic the right align by looking at the .width:
```
    text_level.text = "Level: #{level}"
    text_level.x = Window.width - 5 - text_level.width
```

**Comment by blacktm** [comment]
Yep, that’s all we’ve got so far. I recently prototyped setting an arbitrary anchor point — worked pretty well, so will probably have something soon-ish.

**Comment by blacktm** [comment]
Some basic anchor alignment is now in 9650c7b3c9811bd17d2c6e6205d5660cc9dbf788.
